docs/release-notes/v1.0.1.rst

```diff
@@ -0,0 +1,50 @@
+.. include:: ../references.txt
+
+.. _gammapy_1p0p1_release:
+
+1.0.1 (March 14th, 2023)
+------------------------
+
+Summary
+~~~~~~~
+
+- Released Mar 14th, 2023
+- 9 contributors
+- 30 pull requests since v1.0 (not all listed below)
+
+This is the first bug-fix after v1.0. Several minor bugs and typos in the documentation are corrected.
+
+- A bug in the `~gammapy.dataset.MapEvaluator` was slowing down the model evaluation leading to degraded
+  performance on fitting or flux points estimation. The performances of benchmarks have significantly
+  improved.
+- The "TIMESYS" keyword was not properly exported to the lightcurve table format. This is now corrected.
+- An issue with interpolation in scipy 1.10 is causing problems in gammapy. This specific version is
+  excluded from dependencies.
+
```

Comment:
```suggestion
- Models created from files with a parameter scale different than one were incorrect and this will be fixed by this patch. 
   Note that by default gammapy always serializes the parameters with a scale of one, so this bug affected only files written by hand with a scale different from one.
```